src/sapi/httpd.c

```diff
@@ -258,7 +258,7 @@ balde_sapi_httpd_run(balde_app_t *app)
         g_free(host);
         return 3;
     }
-    g_signal_connect(service, "run", G_CALLBACK(balde_incoming_callback), app);
+    g_signal_connect(service, "incoming", G_CALLBACK(balde_incoming_callback), app);
```

Comment:
This I changed because I could not find any mention about a signal called run, not even on older versions [1].
[1] https://developer.gnome.org/gio/2.34/GSocketService.html

Reply:
that's because the `run` signal is from the threaded socket service, that is what we use: https://developer.gnome.org/gio/stable/GThreadedSocketService.html#GThreadedSocketService-run

Reply:
Ah cool, I assumed as it would be cast to GSocketService it should use its docs and not GThreadedSocketService.